source/state_representation/test/tests/test_parameter.cpp

```diff
@@ -4,18 +4,34 @@
 #include "state_representation/space/cartesian/CartesianState.hpp"
 #include <gtest/gtest.h>
 
+using namespace state_representation;
+
 TEST(ParameterTest, Conversion) {
-  using namespace state_representation;
+  Parameter<int> int_param("test");
+  EXPECT_TRUE(int_param.is_empty());
+  EXPECT_EQ(int_param.get_type(), StateType::PARAMETER_INT);
+  int_param.set_value(2);
+  EXPECT_EQ(int_param.get_value(), 2);
+
+  std::vector<int> values{1, 2, 3};
+  Parameter<std::vector<int>> int_array_param("test", values);
+  EXPECT_FALSE(int_array_param.is_empty());
+  EXPECT_EQ(int_array_param.get_type(), StateType::PARAMETER_INT_ARRAY);
+  for (std::size_t i = 0; i < values.size(); ++i) {
+    EXPECT_EQ(int_array_param.get_value().at(i), values.at(i));
+  }
+//  int_array_param.set_value({1, 2});
```

Comment:
Comment can be removed

Reply:
I left it here because I wanted to ask if this is okay that we can change the size of the parameter array without warning or exception?

Reply:
yeah that's quite alright, it just overwrites the new value